imports/startup/server/vars.js

```diff
@@ -0,0 +1,22 @@
+import { Meteor } from 'meteor/meteor';
+var fs = Npm.require('fs');
+var path = Npm.require('path');
+const vars = {};
+
+const config = JSON.parse(Assets.getText("path.json"));
+vars.pastesNameLenght = 7;
+var __ROOT_APP_PATH__;
+if (config.which === "server") {
+	__ROOT_APP_PATH__ = fs.realpathSync(config.pathAdjustment);
+}else if (config.which === "local") {
+	__ROOT_APP_PATH__ = fs.realpathSync(config.pathAdjustmentTest);
+}
+vars.pastesPath = path.join(__ROOT_APP_PATH__, "/pastes/");
+vars.languages = JSON.parse(fs.readFileSync(path.join(__meteor_bootstrap__.serverDir, "../web.browser/app/languages.json")), "utf8");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier '__meteor_bootstrap__' is not in camel case.](https://www.codacy.com/app/igougi-ui/Mrb-in/file/2618786818/issues/source?bid=2882324&fileBranchId=3313627#l15)